src/index.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback4-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+export * from './providers/server.provider';
+export * from './decorators/grpc-service.decorator';
+export * from './types';
+export * from './keys';
```

Comment:
In the project setup proposed in https://github.com/strongloop/loopback4-extension-starter, we are intentioally not transpiling top-level files (`index.js`, `index.ts`, etc.) to allow the top-level `index.js` to choose a different version (`dist` or `dist6`) depending on the Node.js runtime version (8.x+ or 6.x).

In which case `index.ts` should be a one-line forward to `src/index`, see https://github.com/strongloop/loopback4-extension-starter/blob/3a491d40e7d58ad177484de459050ba127215af7/index.ts#L6-L11

Reply:
@bajtos I'm still not sure I understand this one, that file originally came like that

https://github.com/strongloop/loopback4-extension-starter/blob/3a491d40e7d58ad177484de459050ba127215af7/src/index.ts

So I'm a little confused maybe I'm misunderstanding something here